packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -407,6 +408,12 @@ class ExpansionTile extends StatefulWidget {
   ///   [ExpansionTileThemeData].
   final EdgeInsetsGeometry? childrenPadding;
 
+  /// The color to display behind the expanded children section of the tile.
+  ///
+  /// If this property is null, the value from [ExpansionTileThemeData.childrenPaddingColor] is
+  /// used if specified. If both are null, then [backgroundColor] will be used.
```

Comment:
```suggestion
  /// used. If both are null, then [backgroundColor] will be used.
```

Reply:
We can also add one more comment to explain when this feature is useful:) I thinks this feature is especially useful when the padding is not null and we want the background color of the children tile to be different from the header background.

Reply:
It would be better if we can mention this is only useful when `childrenPadding` is not null.